client/client.go

```diff
@@ -5,6 +5,12 @@ import (
 	wcm "github.com/voidshard/wysteria/common/middleware"
 )
 
+const (
+	FACET_COLLECTION = "collection"
+	FACET_ITEM_TYPE = "itemtype"
+	FACET_ITEM_VARIANT = "variant"
```

Comment:
Just FYI, this naming convention for global const vars isn't consistent with "go idiom". Since this is going to be a 1.0 release, might as well nip it now, right? Naming would be like `FacetCollection`